src/model.rs

```diff
@@ -148,77 +122,54 @@ impl ProofData {
         }
     }
 }
-#[derive(Serialize, Deserialize, Default, PartialEq, Eq, Clone, Encode, Decode)]
-pub struct ProofTransaction {
-    // TODO: investigate if we can remove blob_tx_hash. It can be reconstrustruced from HyleOutput attributes (blob + identity)
-    pub blob_tx_hash: TxHash,
-    pub proof: ProofData,
-    pub contract_name: ContractName,
-}
 
 #[derive(Serialize, Deserialize, Default, PartialEq, Eq, Clone, Encode, Decode)]
-pub struct RecursiveProofTransaction {
-    pub via: ContractName,
+pub struct MultiProofTransaction {
+    pub contract_name: ContractName,
     pub proof: ProofData,
     pub verifies: Vec<(TxHash, ContractName)>,
```

Comment:
linked to previous comment: this was previously (in first node_state impl) a  `Vec<BlobReference>`